src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/logger.html.twig

```diff
@@ -55,27 +55,24 @@
         {% set filters = collector.filters %}
         <div class="log-filters">
             <div id="log-filter-type" class="log-filter">
-                <div class="tab-navigation" role="tablist">
-                    <button role="tab" class="tab-control {{ not has_error_logs and not has_deprecation_logs ? 'active' }}">
-                        <input {{ not has_error_logs and not has_deprecation_logs ? 'checked' }} type="radio" id="filter-log-type-all" name="filter-log-type" value="all">
-                        <label for="filter-log-type-all">All messages</label>
-                    </button>
-
-                    <button role="tab" class="tab-control {{ has_error_logs ? 'active' }}">
-                        <input {{ has_error_logs ? 'checked' }} type="radio" id="filter-log-type-error" name="filter-log-type" value="error">
-                        <label for="filter-log-type-error">
-                            Errors
-                            <span class="badge status-{{ collector.counterrors ? 'error' }}">{{ collector.counterrors|default(0) }}</span>
-                        </label>
-                    </button>
-
-                    <button role="tab" class="tab-control {{ not has_error_logs and has_deprecation_logs ? 'active' }}">
-                        <input {{ not has_error_logs and has_deprecation_logs ? 'checked' }} type="radio" id="filter-log-type-deprecation" name="filter-log-type" value="deprecation">
-                        <label for="filter-log-type-deprecation">
-                            Deprecations
-                            <span class="badge status-{{ collector.countdeprecations ? 'warning' }}">{{ collector.countdeprecations|default(0) }}</span>
-                        </label>
-                    </button>
+                <div class="tab-navigation">
+                    {% set initially_active_tab = has_error_logs ? 'error' : has_deprecation_logs ? 'deprecation' : 'all' %}
+                    <input type="radio" id="filter-log-type-all" name="filter-log-type" value="all" {{ 'all' == initially_active_tab ? 'checked' }}>
+                    <label for="filter-log-type-all" class="tab-control {{ 'all' == initially_active_tab ? 'active' }}">
+                        All messages
+                    </label>
+
+                    <input type="radio" id="filter-log-type-error" name="filter-log-type" value="error" {{ 'error' == initially_active_tab ? 'checked' }}>
+                    <label for="filter-log-type-error" class="tab-control {{ 'error' == initially_active_tab ? 'active' }}">
```

Comment:
I suggest removing the need for the `active` class, allowing to simplify both the template and the JS code. Styling could be done using `.tab-navigation input[type="radio"]:checked + .tab-control`

Reply:
Done